### ISSUE TITLE
[Package Issue]: Libretro.RetroArch reports installer hash mismatch

### ISSUE THREAD

**Issue by tranquilcarbon** [microsoft/winget-pkgs#100030]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installer hash mismatch.

### Brief description of your issue

when attempting to update retroarch, winget reports a "Installer hash does not match; this cannot be overridden when running as admin" error after downloading the installer.

### Steps to reproduce

1. run the command winget upgrade retroarch
2. wait for retroarch to download
3. winget will report a error "Installer hash does not match; this cannot be overridden when running as admin"

### Actual behavior

winget reports "Installer hash does not match; this cannot be overridden when running as admin" after downloading.

### Expected behavior

retroarch to update

### Environment

```raw
Windows Package Manager v1.4.10173
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19045.2728
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.19.10173.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

User Settings: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\settings.json
```

### Screenshots and Logs

[WinGet-2023-03-19-12-51-02.858.log](https://github.com/microsoft/winget-pkgs/files/11011049/WinGet-2023-03-19-12-51-02.858.log)
![Untitled](https://user-images.githubusercontent.com/97110350/226176686-fa2a8841-7667-426f-9570-64cbed5af915.png)

**Comment by krystofkrticka** [comment]
Resolved in #99935

**Comment by mdanish-kh** [comment]
Close with reason: PR merged;
